### PR TITLE
Past project review

### PR DESCRIPTION
**Pull request by ForHemer**
Review a past project

### CODE REVIEW

File: fcc-test-suite-template-master/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body class="container">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
```
Comment (ForHemer) [inline fcc-test-suite-template-master/index.html:13]: Good job using an hx tag here👍 but I'll suggest you to put these 2 lines inside a header tag like this:
<header>
      <h1 id="title">freeCodeCamp Survey Form</h1>
      <p id="description">Thank you for taking the time to help us improve the platform</p>
</header>

File: fcc-test-suite-template-master/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        </div>        
+
+        <p class="center">Which option best describes your current role?</p>
+        <div class="selectContainer">
+          <select id="dropdown" name="s">
+            <option selected disabled value="">Select current role</option>
+            <option value="">Student</option>
+            <option value="">Full time job</option>
+            <option value="">Full time lerner</option>
+            <option value="">Preder not to say</option>
```
Comment (ForHemer) [inline fcc-test-suite-template-master/index.html:38]: It's important to specify the content of the **value** attribute

**Review by ForHemer — COMMENTED** [review]
This is my comment about one of my past projects.

File: fcc-test-suite-template-master/index.html (hunk 1 above)
Comment (MrBrN197) [reply fcc-test-suite-template-master/index.html:13]: - [ ] Kindly try to give a reason for your suggestion. Sometimes it's not clear why you are suggesting a change so it's best to elaborate on the reasoning for your suggestion.

File: fcc-test-suite-template-master/index.html (hunk 2 above)
Comment (MrBrN197) [reply fcc-test-suite-template-master/index.html:38]: - [ ] Kindly try to give a reason for your suggestion. Sometimes it's not clear why you are suggesting a change so it's best to elaborate on the reasoning for your suggestion.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @ForHemer 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — APPROVED** [review]
## Hi, @ForHemer  👋
Thanks for making the changes I suggested.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**